contrib/babelfishpg_tsql/sql/sys_function_helpers.sql

```diff
@@ -4,7 +4,7 @@
 CREATE OR REPLACE FUNCTION sys.babelfish_get_last_identity()
 RETURNS INT8
 AS 'babelfishpg_tsql', 'get_last_identity'
-LANGUAGE C;
+LANGUAGE C STABLE;
```

Comment:
have we verified the behaviour for @@identity within a statement? is it stable where every row will see the same value? not if we fetch global value again and again?